extensions/wikia/GlobalNavigation/GlobalNavigationAccountNavigationController.class.php

```diff
@@ -129,6 +132,21 @@ private function renderPersonalUrl( $id, $openingTagOnly = false ) {
 		return $markup;
 	}
 
+	public function getAuthOptions() {
+		return wfMessage( 'global-navigation-account-navigation-options', [
+			$this->app->renderView('GlobalNavigation', 'authLink', [
```

Comment:
Creating new view and controller for rendering one link sounds link an overkill.

Maybe we could use ```XML::element```? Example here: https://github.com/Wikia/app/blob/dev/includes/specials/SpecialVersion.php#L82